aodndata/soop/soop_xbt_nrt.py

```diff
@@ -360,28 +360,18 @@ def netcdf_writer(profile_data, output_dir):
         for var in ['TEMP_quality_control', 'DEPTH_quality_control']:
             template.variables[var][att] = np.int8(template.variables[var][att])
 
-    time_attr_format = '%Y-%m-%dT%H:%M:%SZ'
-    time_coverage_start = datetime.datetime.strftime(profile_data['profile_geotime']['date_utc'], time_attr_format)
-    time_coverage_end = time_coverage_start
+    template.add_extent_attributes()
+    template.add_date_created_attribute()
 
     abstract = get_info_xbt_config(profile_data['profile_metadata']['XBT_line'])['abstract']
     title = get_info_xbt_config(profile_data['profile_metadata']['XBT_line'])['title']
 
     template.global_attributes.update({
         'abstract': abstract,
         'title': title,
-        'time_coverage_start': time_coverage_start,
-        'time_coverage_end': time_coverage_end,
         'featureType': 'profile',
-        'date_created': datetime.datetime.utcnow().strftime(time_attr_format),
-        'geospatial_lat_min': profile_data['profile_geotime']['latitude'],
-        'geospatial_lat_max': profile_data['profile_geotime']['latitude'],
-        'geospatial_lon_min': profile_data['profile_geotime']['longitude'],
-        'geospatial_lon_max': profile_data['profile_geotime']['longitude'],
-        'geospatial_vertical_min': profile_data['profile_data']['depth'].min(),
-        'geospatial_vertical_max': profile_data['profile_data']['depth'].max(),
         'history': "File created {date_created}".format(
-            date_created=datetime.datetime.utcnow().strftime(time_attr_format))
+            date_created=datetime.datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ'))
```

Comment:
Since you've already set the creation date once, why not reuse that value? i.e.
```python
    date_created=template.global_attributes['date_created']
```

Also, we usually put the date first in a `history` attribute: `"{date_created}: file created"`

Reply:
I followed what had been done for soop tmv
https://github.com/aodn/python-aodndata/blob/master/aodndata/soop/soop_tmv_nrt.py#L196

I guess this should be changed as well